.gitignore

```
@@ -1,7 +1,3 @@
 .*.swp
-bin
-pkg
 
-vendor/**/.gitignore
-vendor/**/.idea/
-vendor/**/.travis.yml
+matrix-federation-tester
```

.gitignore entries should be anchored if you are matching a specific path:

```suggestion
/matrix-federation-tester
```